controllers/dnschaos/types.go

```diff
@@ -61,7 +61,7 @@ func (r *endpoint) Apply(ctx context.Context, req ctrl.Request, chaos v1alpha1.I
 		return err
 	}
 
-	pods, err := selector.SelectAndFilterPods(ctx, r.Client, r.Reader, &dnschaos.Spec, config.ControllerCfg.ClusterScoped, config.ControllerCfg.TargetNamespace, config.ControllerCfg.AllowedNamespaces, config.ControllerCfg.IgnoredNamespaces)
+	pods, err := selector.SelectAndFilterPods(ctx, r.Client, r.Reader, &dnschaos.Spec, config.ControllerCfg.ClusterScoped, config.ControllerCfg.TargetNamespace, config.ControllerCfg.EnableFilterNamespace)
```

Comment:
I'm not sure this compatibility break be merged into main or some other branch

Reply:
`master` will be released as `1.2.0`, which will be safe for compatibility break.

@cwen0 Will we bring this feature with `1.2.0`?

Reply:
Yep